### ISSUE TITLE
Long hairstyles

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9969]
**Preferred term label:**
Long hairstyles

**Synonyms**

**Definition (free text, please give PubMed ID)**
Long hair, full beard, full grown pubic hair

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Black piedra

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001089
- label: Long hairstyle
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:31424762
- def: A past medical history of having long, full hair.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9969
- synonyms:
